doc/source/release-notes/1.5.3.rst

```diff
@@ -0,0 +1,26 @@
+.. _1.5.3:
+
+================================================================================
+Percona Monitoring and Management |release|
+================================================================================
+
+:Date: December 15, 2017
+
+|percona| announces the release of |pmm.name|
+|release|. This release contains fixes to bugs found after |pmm.name|
+|prev-release| was released.
+
```

Comment:
.. and also it will include some important fixes and improvements not related to  Prev. Release